src/index.ts

```diff
@@ -97,7 +97,7 @@ interface Options {
 }
 
 export const replacer = function replacer(options: Options) {
-  let objects: { keys: string; value: any }[];
+  let objects: Map<any, string>;
```

Comment:
What about IE11?

Reply:
Can we polyfill it?

Reply:
Yeah the polyfill looks like this:
https://polyfill.io/v3/polyfill.js?features=Map&flags=always

Reply:
asking consumers to polyfill is better than bundling one (might be what you mean to do already, of course 🙂)

Reply:
I'm leaning in that direction as well @SimenB !

Reply:
I'm good with asking users to polyfill @SimenB @ndelangen

Reply:
Shall we push this forwards as a new major?

Reply:
I think we should but can we talk through what the docs would look like first? Will we polyfill in storybook or ask the user to polyfill in their installation of storybook? (Maybe we already do?)

Here's React's docs

https://reactjs.org/docs/javascript-environment-requirements.html